travis-ci/vmtest/run.sh

```diff
@@ -425,7 +425,6 @@ sudo chmod 755 "$mnt/etc/rcS.d/S50-run-tests"
 
 poweroff_script="#!/bin/sh
 
-echo travis_fold:start:shutdown
```

Comment:
it's supposed to match https://github.com/libbpf/libbpf/pull/324/files#diff-0b57a502f172fb2b6b4d53d5c3f0f9154af42788f3444c1224ab1d3b51d9ce62L457, but we don't have travis_fold function inside the VM.. I see how that's painful now, so not sure what to do about it. We can drop it or just add (uncoditionally) both travis and github actions-style section start output